xml/gl.xml

```diff
@@ -42242,7 +42242,7 @@ typedef unsigned int GLhandleARB;
                 <!-- <command name="glTexturePageCommitmentEXT"/> -->
             </require>
         </extension>
-        <extension name="GL_EXT_sparse_texture2" supported="gl"/>
+        <extension name="GL_EXT_sparse_texture2" supported="gl|gles2"/>
```

Comment:
The VK-GL-CTS drives what function pointers are loaded for the required functions off of this as well. For GL entry points it only loads them if they have 'glcore' specified.